package.json

```diff
@@ -4,11 +4,18 @@
   "description": "Deal/Checkout modal for coupon app",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest",
+    "test:watch": "jest --watch --coverage --colors",
+    "test:coverage": "jest --coverage --colors",
     "start": "nodemon server",
     "build": "webpack -d -w",
     "seed": "cd db/ && npx sequelize db:drop && npx sequelize db:create && npx sequelize db:migrate && npx sequelize db:seed:all || npm run fallback",
-    "fallback": "cd db/ && npx sequelize db:create && npx sequelize db:migrate && npx sequelize db:seed:all"
+    "fallback": "cd db/ && npx sequelize db:create && npx sequelize db:migrate && npx sequelize db:seed:all",
+    "jest": {
+      "transform": {
```

Comment:
consider moving this into the `jest.config.json` file (though it can be in either).